src/command.test.js

```diff
@@ -13,12 +13,20 @@ test(`converts unix-style env variable usage for windows`, () => {
 test(`leaves command unchanged when not a variable`, () => {
   expect(commandConvert('test')).toBe('test')
 })
+
 test(`converts windows-style env variable usage for linux`, () => {
   isWindowsMock.__mock.returnValue = false
   expect(commandConvert('%test%')).toBe('$test')
 })
 
 test(`leaves variable unchanged when using correct operating system`, () => {
-  isWindowsMock.__mock.returnValue = true
+  isWindowsMock.__mock.returnValue = false
   expect(commandConvert('$test')).toBe('$test')
 })
+
+test(`is stateless`, () => {
+  // this test prevents falling into regexp traps like this:
+  // http://bit.ly/1zWb3tn
```

Comment:
Sure! Tried to use http://stackoverflow.com/questions/1520800/why-regexp-with-global-flag-in-javascript-give-wrong-results/1520853#1520853 but it was too long for the 80 chars limit.